components/brave_shields/browser/ad_block_service_browsertest.cc

```diff
@@ -976,3 +976,42 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, CosmeticFilteringUnhide) {
               &as_expected));
   EXPECT_TRUE(as_expected);
 }
+
+// Test scriptlet injection that modifies window attributes
+IN_PROC_BROWSER_TEST_F(AdBlockServiceTest,
+                       CosmeticFilteringWindowScriptlet) {
+  /* "content" below corresponds to the following scriptlet:
+   * ```
+   * (function() {
+   *   const send = window.getComputedStyle;
+   *   window.getComputedStyle = function(selector) {
+   *     return { 'color': 'Impossible value' };
+   *   }
+   * })();
+   * ```
```

Comment:
Better to just base64 encode the actual value for next time `base::Base64Encode`.